index.html

```diff
@@ -47,7 +47,126 @@ <h1 class="title">Hello There! 👋 <br /> Welcome to my Website!</h1>
+        <img alt="work-item-img" src="./assets/img/work/work-3-cover.png">
+        <h3 class="work-title">Tonic</h3>
+        <div class="work-subtitle">
+          <span>Canopy</span>
+          <span>Back End Dev</span>
+          <span>2015</span>
+        </div>
+        <p class="work-details">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="work-technologies">
+          <li class="technology-tag">html</li>
+          <li class="technology-tag">css</li>
+          <li class="technology-tag">javascript</li>
+        </ul>
+        <button class="view-work-btn">See Project</button>
+      </div>
+      <div class="work-item">
+        <img alt="work-item-img" src="./assets/img/work/work-4-cover.png">
+        <h3 class="work-title">Multi-Post Stories</h3>
+        <div class="work-subtitle">
+          <span>Canopy</span>
+          <span>Back End Dev</span>
+          <span>2015</span>
+        </div>
+        <p class="work-details">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="work-technologies">
+          <li class="technology-tag">html</li>
+          <li class="technology-tag">css</li>
+          <li class="technology-tag">javascript</li>
+        </ul>
+        <button class="view-work-btn">See Project</button>
+      </div>
+    </section>
+    <section id="aboutMe" class="about-me">
+      <div class="top">
+        <h2 class="title">About Myself</h2>
+        <p class="introduction">I'm Joshua Ivie and I'm a designer and full stack engineer with 5+ years of experience.
+          Like anything you
+          see or just want to chat over a coffee? Here are a few links you can contact me
+          through.</p>
+        <a href="#" class="sub-title">Lets Connect</a>
+        <ul class="social-links">
+          <li class="link"><a href="https://twitter.com/joshuaivie_" target="blank"><i class="fa fa-twitter"></i></a>
+          </li>
+          <li class="link"><a href="https://www.linkedin.com/in/joshuaivie/" target="blank"><i
+                class="fa fa-linkedin"></i></a></li>
+          <li class="link"><a href="https://github.com/joshuaivie" target="blank"><i class="fa fa-github"></i></a></li>
+        </ul>
 
+        <button class="resume-btn">Get my resume</button>
+      </div>
+      <div class="bottom">
+        <ul class="technical-knowledge">
+          <li>
+            <div class="knowledge-area active">
+              <h3 class="title">Languages</h3>
+              <i class="fa fa-chevron-down"></i>
+            </div>
+            <ul class="technical-knowledge-list">
+              <li><img src="./assets/img/technology/javascript.png" alt="javascript">Javascript</li>
+              <li><img src="./assets/img/technology/html.png" alt="html">HTML</li>
+              <li><img src="./assets/img/technology/css.png" alt="css">CSS</li>
+            </ul>

```

Comment:
- [ ]  Great job working on this part of the project, but unfortunately language titles are not aligned with specific icons as you can see in the screenshots below. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. 
 - [ ] Please write `JavaScript` instead of `Javascript` those little details can have a big impact in your future career as a web developer 😄 

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/148662367-677cd5df-35b9-4a1a-b70a-533ed706bf27.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/148662374-1916a4d7-a97c-4610-b756-f50598ea32f5.PNG)